app/assets/javascripts/components/ops/tenant-component.js

```diff
@@ -0,0 +1,76 @@
+ManageIQ.angular.app.component('tenantComponent', {
+  bindings: {
+    id: '=?',
+    divisible: '=',
+    parent: '=?',
+    angularForm: '=',
```

Comment:
`angularForm` certainly doesn't belong here, since it is declared *in* that template..